nixos/modules/services/misc/sssd.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.sssd;
+  sssdConf = pkgs.writeText "sssd/sssd.conf" cfg.sssdConf;
```

Comment:
Thanks for catching that. That was a leftover from when I was considering allowing the sssd.conf file to be entered into configuration.nix. I decided against it since sssd.conf is only readable by root, but I forgot to remove the line.